src/PendingCalls/TestCall.php

```diff
@@ -403,6 +403,16 @@ public function throwsNoExceptions(): self
         return $this;
     }
 
+    /**
+     * Registers a closure to be called after the test has been executed.
+     */
+    public function after(Closure $after): self
```

Comment:
It would make sense to support any valid callback, not just closures, for instance:
- Closure
- Callable class with `__invoke()` method
- Array callable syntax: `[$instance, 'method']`, `['ClassName', 'staticMethod']`
- Function callable: `'function_name'`

It can be implemented using `callable` pseudo-type annotation, for example:
```php
public function after(callable $callback): self
{
    $this->testCaseMethod->after = Closure::fromCallable($callback);
}
```

Reply:
Responding to all your reviews here. Thanks for taking the time to leave the great feedback! As this PR was already approved I'm going to hold off on making any changes until I hear more from the maintainers on which route they want to go in. Again, thanks for taking the time, I really appreciate it!